gameofgame.py

```diff
@@ -86,15 +90,20 @@ def check_guess(guess, country):
         print("You both should learn about capital cities in the world!!!")
         
     # Print the current scores
-    print(f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}")
+    print(
+        f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}"
+        )
     
     # Ask if the players want to continue playing
     play_again = input("Do you want to play again? (y/n) ")
-    if play_again.lower() != "y":
+    if play_again.lower() != "n":
         break
-    
-else:
```

Comment:
You have lost this `else` statement in the current version of the code. This will cause to print the tie message even when the players want to keep playing.